docs/conf.py

```diff
@@ -139,11 +141,14 @@ def setup(app):
         # The glob patterns in `babel_ocds_codelist.cfg` should match these.
         (glob(str(project_dir / 'codelists' / '*.csv')), project_build_dir / 'codelists', codelists_domain),
         (glob(str(project_dir / 'codelists' / '*.csv')), language_dir / 'codelists', codelists_domain),
+        # The glob patterns in `babel_ocds_mapping.cfg` should match these filenames.
+        (glob(str(basedir / 'mapping' / '*.csv')), project_build_dir, mapping_domain),
+        (glob(str(basedir / 'mapping' / '*.csv')), language_dir, mapping_domain),
     ], localedir, language, headers, version=branch)
 
     # Copy our mapping files as well. This currently does not perform translation, which would need to be added.
-    for filename in glob(str(basedir / 'mapping' / '*.csv')):
-        with open(filename) as f:
-            mapping_file = f.read()
-        with open(language_dir / os.path.basename(filename), 'w') as f:
-            f.write(mapping_file)
+    # for filename in glob(str(basedir / 'mapping' / '*.csv')):
+    #    with open(filename) as f:
+    #        mapping_file = f.read()
+    #    with open(language_dir / os.path.basename(filename), 'w') as f:
+    #        f.write(mapping_file)
```

Comment:
Let's delete these lines, since those files are now controlled by the translation mechanism.

Reply:
Removed in https://github.com/open-contracting/infrastructure/pull/225/commits/caa60094f8364affe51c460d7d6cef23b54dba87